rosidl_cli/rosidl_cli/command/generate/api.py

```diff
@@ -87,15 +88,39 @@ def generate(
     else:
         os.makedirs(output_path, exist_ok=True)
 
-    if len(extensions) > 1:
-        return [
+    def extra_kwargs(func: Callable, **kwargs: Any) -> Dict[str, Any]:
```

Comment:
just a comment that i understand correctly.
this dynamically passes additional arguments (type_description_files) to the generate method of extensions.
This ensures that only the keyword arguments valid for the generate method of a given extension are passed, avoiding runtime errors caused by unexpected or unsupported arguments.

Reply:
Correct! Our intent is to backport this to Jazzy, so it's backwards compatible.